### PR TITLE
Add 2019 group photo

### PR DESCRIPTION
**Pull request by wtbarnes**
This PR adds the group photo from the 2019 meeting and changes all of the `figure` tags to `image` for the group photos on the main page as the former was not centering the photos for some unknown reason.

### CODE REVIEW

**Comment by wtbarnes** [comment]
@mbobra or @SolarDrew, do you know who took the 2019 photo? I've currently left a placeholder for the copyright holder of the image

**Comment by mbobra** [comment]
I'll inquire and let you know :)

**Comment by mbobra** [comment]
📷 credit goes to Pam Jeffries!

File: pages/index.rst [hunk 1]
```diff
@@ -21,6 +21,11 @@ The 2019 Python in Astronomy conference was held at the
 `Space Telescope Science Institute </http://www.stsci.edu/>`__ in Baltimore, MD USA
 on July 29th - August 2nd. More details can be found `here </2019>`__.
 
+.. image:: /images/pyastro19_attendees.jpg
+   :alt: Python in Astronomy 2019 Attendees, Copyright 2019 <??????>
```
Comment (SolarDrew) [inline pages/index.rst]:
```suggestion
   :alt: Python in Astronomy 2019 Attendees, Copyright 2019 Pam Jeffries
```

**Pull request merged**